scripts/docs-collator/utils/rendering.py

```diff
@@ -79,3 +75,27 @@ def parse_github_action_repo_name(name):
     action_name = '-'.join(name_items[2:])
 
     return action_name
+
+
+def replace_relative_links_with_github_links(repo, content, relative_path=None):
+    links = re.findall(RELATIVE_LINK_PATTERN, content)
+
+    for link in links:
+        # ignore links to images, anchors and emails
+        if link.startswith('images/') or link.startswith('#') or link.startswith('mailto:'):
+            continue
+
+        updated_link = link
+
+        # remove leading './' or '/'
+        if link.startswith('./'):
+            updated_link = updated_link.replace('./', '', 1)
+        elif link.startswith('/'):
+            updated_link = updated_link.replace('/', '', 1)
```

Comment:
This feels wrong. We're taking an absolute path and making it relative. Did you see any examples of where this would be required?

Reply:
actually it is the other way around. We are taking relative link and replacing them with absolute path links to github:

So this `examples/complete` gets replaced with `https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment/tree/main/examples/complete`

Complete log or what gets replaced with what can be seen here: https://gist.github.com/zdmytriv/3e42f497539b16a786710e01e067fbd7

Reply:
ok i got what you mean. There are two modules that have [full path](https://github.com/cloudposse/terraform-aws-sso/blob/main/README.yaml#L93).

We can't have absolute paths neither on GitHub nor in docs. Github renders `README.html` from `README.md` by making absolute path as relative path to root folder of the repo. This is what we do as well in docs.

I think it is fine to have it like this because we will never run into link that will have absolute path like '/cloudposse/terrraform-aws-sso/...' in `README.md`.

Reply:
Ok, let's try it out.